tests/unittest/_torch/ray_orchestrator/multi_gpu/test_llm_update_weights_multi_gpu.py

```
@@ -579,3 +584,166 @@ def filter_fn(name: str) -> bool:
+        # Finalize once to trigger post_load_weights on all modules.
+        llm._collective_rpc("update_weights", (None,))
+
+        llm_logits, ref_logits = run_generate(llm, hf_model, prompts, sampling_params)
+        compare_logits(llm_logits, ref_logits)
+
+
+def _nemotron_h_subprocess_entry(result_queue):
+    try:
+        _nemotron_h_body()
+        result_queue.put(None)
+    except BaseException:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `BaseException`

(BLE001)